CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - 
 - Lambda transform ([#762](https://github.com/tinkoff-ai/etna/issues/762))
-- 
+- assemble pipelines ([#717](https://github.com/tinkoff-ai/etna/issues/717))
```

Comment:
We usually add PRs to CHANGELOG.md, not Issues, change it here to https://github.com/tinkoff-ai/etna/pull/774.